.github/workflows/android-perf.yml

```diff
@@ -0,0 +1,198 @@
+name: android-perf
+
+on:
+  schedule:
+    - cron: 0 0 * * *
+  push:
+    tags:
+      - ciflow/perf-android
+  pull_request:
+  # Note: GitHub has an upper limit of 10 inputs
+  workflow_dispatch:
+    inputs:
+      models:
+        description: Models to be benchmarked
+        required: false
+        type: string
+        default: stories110M
+      devices:
+        description: Target devices to run benchmark
+        required: false
+        type: string
+        default: false
+      delegates:
+        description: Backend delegates
+        required: false
+        type: string
+        default: xnnpack
+      threadpool:
+        description: Run with threadpool?
+        required: false
+        type: boolean
+        default: false
+      benchmark_configs:
+        description: The list of configs used the benchmark
+        required: false
+        type: string
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref_name }}-${{ github.ref_type == 'branch' && github.sha }}-${{ github.event_name == 'workflow_dispatch' }}-${{ github.event_name == 'schedule' }}
+  cancel-in-progress: true
+
+permissions: read-all
+
+jobs:
+  set-models:
+    runs-on: linux.2xlarge
+    outputs:
+      models: ${{ steps.set-models.outputs.models }}
+    steps:
+      - name: Set models
+        id: set-models
+        shell: bash
+        run: |
+          set -ex        
+          MODELS="${{ inputs.models || 'stories110M' }}"
+          echo "models=$(echo $MODELS | jq -Rc 'split(",")')" >> $GITHUB_OUTPUT
```

Comment:
Finally get this one working correctly :)

Reply:
@huydhn Oh cool! It hack is for testing on PR only and should revert it before merging right?

Reply:
It's fine to keep the current version I think, `set -ex` is just to print the command to console.

Reply:
@huydhn I'm talking about this line:
```
MODELS="${{ inputs.models || 'stories110M' }}"
```

I suspect it works on PR because the default value "stories110M" is passed here instead of retrieving from the `inputs.models`. It's fine if it's to make it testable it on PR. When we dispatch the workflow from GA UI, I expect it to retrieve the default value from the `inputs.models`, i.e. changing it to back `MODELS="${{ inputs.models }}"` should work, and we should probably merge it like that.

Reply:
I think we need it there as the default value because I don't think `ciflow` counts as a workflow dispatch.  The gotcha here is that the list of inputs you have is only for workflow dispatch, so the model name, for example, is null in other cases.

Here is an example to illustrate this https://github.com/pytorch/builder/blob/main/.github/workflows/validate-binaries.yml has both `workflow_call` and `workflow_dispatch`.  Each has a the same list of inputs because they are different events.

Reply:
> I think we need it there as the default value because I don't think ciflow counts as a workflow dispatch.

If it doesn't work with `ciflow`, we can remove it. It's okay to only allow schedule on-demand benchmarking from workflow dispatcher UI. Users won't be able to pass any argument via `ciflow`, so tagging `ciflow` will just run the default model which may not be helpful.

Reply:
Removed `ciflow`. Restored `MODELS="${{ inputs.models || 'stories110M' }}"` to `MODELS="${{ inputs.models }}` since we don't need to trigger via `ciflow`. Expect it will read the default values from inputs.models when scheduling from workflow_dispatcher UI